### ISSUE TITLE
 2.4.3 Menu- Student- Accessibility Fixes - Level A - Focus Order

### ISSUE THREAD

**Issue by michesmith** [bcgov/SIMS#3141]
**User Story**
**As a student with accessibly constraints , I need to be able to tab through menu items to complete actions related to my application. 

**Acceptance Criteria**

- [ ] In Student "Applications" - "Application Actions", change tabindex for menu items to "0". User should be able to tab to button, enter drop down with "Space" or "Enter" and then tab through options. 

<img width="1334" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN25HQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c3435c7c37a793ecb00cd6cb39a3bbf78ad4948e/image.png" alt="image.png" />

**Context:**
- These issues were uncovered through accessibility audit via CITZ. 
- Detailed Spreadsheet for Student s inserted inline and also attached here: [2024_Student Application_Audit.xlsx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN0xsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--578821d4addd12c1babcafba5c861e1f5456bc9c/2024_Student%20Application_Audit.xlsx)
- If a web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability.

**The Problem**
- Tab index attributes of -2 don't allow intuitive tab interactions.

**Technical**

- [x] Check how to change the current dropdown menu to allow tab navigation across all its subitems.
- [x] Implement the solution in the application tracker landing page present in the image above.
